go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prebid/prebid-server/v2
 
-go 1.20
+go 1.21
```

Comment:
To update the version of Go we must also update the build pipeline (hence the build failures). That's better suited for a standalone PR. Please keep with targeting 1.19 and 1.20 here. Instead of `maps.Clone`, we have a `maputil.Clone` method.